src/java.base/share/conf/security/java.security

```diff
@@ -28,6 +28,45 @@
 # Properties in this file are typically parsed only once. If any of the
 # properties are modified, applications should be restarted to ensure the
 # changes are properly reflected.
+#
+# The special "include" property can be defined one or multiple times in
+# this file with a filesystem path value. The effect of each definition
+# is to include a referred security properties file inline, adding all
+# its properties and values. Included files, as well as files pointed by
```

Comment:
Not sure if "properties and values" are precise. A "property" already contains its "key" and "value", right?

Reply:
Ok